src/pages/RepoPage/CoverageOnboarding/ExampleBlurb/ExampleBlurb.tsx

```diff
@@ -3,22 +3,23 @@ import A from 'ui/A'
 const ExampleBlurb = () => {
   return (
     <div data-testid="example-blurb">
-      &#128193; See an example{' '}
+      &#128193; View a{' '}
       <A
-        to={{ pageName: 'codecovExampleWorkflow' }}
+        to={{ pageName: 'codecovExampleJSCircleCIWorkflow' }}
         isExternal
         hook="codecov-workflow-intro"
       >
-        repo here
+        javascript config.yml example
```

Comment:
nit: `javascript` -> `JavaScript`

Reply:
oops, accidentally forgot to make this change. I'll add it to another PR